spec/models/chess_piece_spec.rb

```diff
@@ -37,4 +37,20 @@
     end
   end
 
+  describe "#is_obstructed?" do
+    context "given a piece at [4, 4]" do
+      let(:piece) {ChessPiece.create(position_x: 4, position_y: 4)}
```

Comment:
if you look at your method, `vertically_obstructed?` is calling another method called `game`. yet this `ChessPiece` does not `belong_to` any game. It's `game_id` is `nil`. Therefore when you can `piece.game` you will get `nil`. You'll have to create a `game` record first and then assign these pieces to the same game.

```
let(:game) { Game.create }
let(:piece) {ChessPiece.create(position_x: 4, position_y: 4, game: game)}
let(:second_piece) {ChessPiece.create(position_x: 4, position_y: 6, game: game)}
```